src/app/showcase/doc/rating/templatedoc.ts

```diff
@@ -9,7 +9,7 @@ import { Code } from '../../domain/code';
         </app-docsectiontext>
         <div class="card flex justify-content-center">
             <p-rating [(ngModel)]="value" stars="5">
-                <ng-template pTemplate="cancel">
+                <ng-template pTemplate="cancelicon">
```

Comment:
If you change this part, also you should add `cancelicon` in the code JSON where you have the examples:

![image](https://github.com/primefaces/primeng/assets/19764334/dd7b9763-f1ad-4f2a-8fe1-6d684a416c93)

Reply:
Ah sorry never saw that, its done now

Reply:
> Ah sorry never saw that, its done now

No problem, thanks for your contribution!